src/UI/Components/OTPinput.tsx

```diff
@@ -0,0 +1,75 @@
+// OTPInputs.tsx
+
+import React, {useRef} from 'react';
+import {StyleSheet, TextInput, View} from 'react-native';
+
+interface OTPInputsProps {
+  otp: string[];
+  handleOtpChange: (index: number, value: string) => void;
+  isChecked: boolean;
+}
+
+const OTPInputs: React.FC<OTPInputsProps> = ({
```

Comment:
do not use React.FC, let us just have a Arrow function taking otpInputsProps

Reply:
+documentation

Reply:
Done